Dockerfile

```diff
@@ -1,31 +1,11 @@
-FROM python:3.7.6-alpine3.10
-
-RUN apk add --no-cache \
-        bash \
-        ca-certificates \
-        g++ \
-        gcc \
-        git \
-        libressl \
-        libressl-dev \
-        linux-headers \
-        make \
-        musl-dev \
-        zlib-dev
-
-ARG LIBRDKAFKA_VERSION=1.0.1
-RUN wget https://github.com/edenhill/librdkafka/archive/v$LIBRDKAFKA_VERSION.tar.gz \
-        -O /tmp/librdkafka-$LIBRDKAFKA_VERSION.tar.gz && \
-    cd /tmp/ && \
-    tar zxf librdkafka-$LIBRDKAFKA_VERSION.tar.gz && \
-    cd librdkafka-$LIBRDKAFKA_VERSION && \
-    ./configure && \
-    make && \
-    make install && \
-    cd .. && rm -fr librdkafka-$LIBRDKAFKA_VERSION
-
+FROM python:3.8-slim-buster
+RUN apt-get update && apt-get install -y sudo && \
+    echo "app ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/app && \
+    useradd -d /app --create-home app && \
+    rm -rf /var/lib/apt/lists/*
 COPY requirements.txt /tmp/
 RUN pip install --no-cache-dir -r /tmp/requirements.txt
-
+WORKDIR /app
+USER app
```

Comment:
Nice!

Reply:
Why do you need sudo inside a docker container ?

Reply:
@pythops Installing dependencies with `apt`

Reply:
As we discussed earlier, the user can be added in the service image in which case you don't to install sudo and therefore avoid any vulnerabilities related to. We can then update the Dockerfile in the service template and include the user there.